docs-en/chapter_data_structure/classification_of_data_structure.md

```diff
@@ -20,23 +21,28 @@ Non-linear data structures can be further divided into tree structures and netwo
 
 ## Physical Structure: Contiguous and Dispersed
 
-**When an algorithm program runs, the data being processed is mainly stored in memory**. The following figure shows a computer memory stick, each black block containing a memory space. We can imagine memory as a huge Excel spreadsheet, where each cell can store a certain amount of data.
 
-**The system accesses data at the target location through memory addresses**. As shown in the figure below, the computer allocates numbers to each cell in the table according to specific rules, ensuring each memory space has a unique memory address. With these addresses, programs can access data in memory.
+**During the execution of an algorithm, the data being processed is stored in memory**. The figure below shows a computer memory stick where each black square represents a memory space. We can imagine memory as a vast Excel spreadsheet in which each cell can store data of a specific size.
+
+**The system accesses the data at the target location by means of a memory address**. As shown in the figure below, the computer assigns a unique identifier to each cell in the table according to specific rules, ensuring that each memory space has a unique memory address. With these addresses, the program can access the data stored in memory.
+
 
 ![Memory Stick, Memory Spaces, Memory Addresses](classification_of_data_structure.assets/computer_memory_location.png)
 
 !!! tip
 
-    It's worth noting that comparing memory to an Excel spreadsheet is a simplified analogy. The actual working mechanism of memory is more complex, involving concepts like address space, memory management, cache mechanisms, virtual memory, and physical memory.
 
-Memory is a shared resource for all programs. When a block of memory is occupied by one program, it cannot be used by others simultaneously. **Therefore, memory resources are an important consideration in the design of data structures and algorithms**. For example, the peak memory usage of an algorithm should not exceed the system's remaining free memory. If there is a lack of contiguous large memory spaces, the chosen data structure must be able to store data in dispersed memory spaces.
+    It is worth noting that comparing memory to an Excel spreadsheet is a simplified analogy. The actual memory working mechanism is more complex, involving concepts such as address, space, memory management, cache mechanism, virtual and physical memory.
```

Comment:
I think the original sentence is better.

It's worth noting that comparing memory to an Excel spreadsheet is a simplified analogy. The actual working mechanism of memory is more complex, involving concepts like address space, memory management, cache mechanisms, virtual and physical memory.